app/src/main/res/layout/list_item_teleconsult_medicine.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content">
+
+  <TextView
+    android:id="@+id/medicineNameTextView"
+    style="@style/Clinic.V2.TextAppearance.Body1Left.Grey0"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:text="Amlodipine" />
+
+  <androidx.constraintlayout.widget.Barrier
```

Comment:
I'm curious, what does a `Barrier` do differently from just setting a margin?

Reply:
Barrier will allow us to set a common margin for the referenced ids instead of manually setting the margins for each view separately.

Reply:
Ah, I see. TIL.